clang/test/AST/HLSL/StructuredBuffer-AST.hlsl

```diff
@@ -1,64 +1,64 @@
-// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-library -x hlsl -ast-dump -DEMPTY %s | FileCheck -check-prefix=EMPTY %s
```

Comment:
hmm....maybe a line endings change issue on this file?  I'm wondering if some config in the root of this repo has changed since I've heard other people mention line endings getting messed up?

Reply:
Yes, line endings seem to be messed up. I've checked out main for both these files and only changed the necessary parts,  and pushed, but there must be some automatic thing my editor is doing to mess up the line endings.

Reply:
We should try to fix this before completing the PR.

Reply:
Fixed!